packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -210,15 +210,15 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         controller: _tabController,
         isScrollable: true,
         onTap: _pushScreenToLocalPageRoute,
-        tabs: [for (var screen in widget.tabs) screen.buildTab(context)],
+        tabs: [for (var screen in widget.tabs) screen.buildTabWidget(context)],
```

Comment:
nit: so many things are widgets that is probably best to not prepend method names with the word widget.
For example, it is a `build` method not a `buildWidget` method.

Reply:
gotcha, will rename